whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -44,31 +51,21 @@
 import org.springframework.stereotype.Component;
 
 import javax.annotation.Nullable;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
+import javax.ws.rs.InternalServerErrorException;
+import javax.ws.rs.core.Response;
 import java.util.Map;
+import java.util.List;
+import java.util.Iterator;
 import java.util.Set;
+import java.util.HashMap;
 
-import static net.ripe.db.whois.common.rpsl.AttributeType.ADDRESS;
-import static net.ripe.db.whois.common.rpsl.AttributeType.ADMIN_C;
-import static net.ripe.db.whois.common.rpsl.AttributeType.DS_RDATA;
-import static net.ripe.db.whois.common.rpsl.AttributeType.E_MAIL;
-import static net.ripe.db.whois.common.rpsl.AttributeType.FAX_NO;
-import static net.ripe.db.whois.common.rpsl.AttributeType.GEOLOC;
-import static net.ripe.db.whois.common.rpsl.AttributeType.MNT_BY;
-import static net.ripe.db.whois.common.rpsl.AttributeType.ORG;
-import static net.ripe.db.whois.common.rpsl.AttributeType.ORG_NAME;
-import static net.ripe.db.whois.common.rpsl.AttributeType.PERSON;
-import static net.ripe.db.whois.common.rpsl.AttributeType.PHONE;
-import static net.ripe.db.whois.common.rpsl.AttributeType.ROLE;
-import static net.ripe.db.whois.common.rpsl.AttributeType.TECH_C;
-import static net.ripe.db.whois.common.rpsl.AttributeType.ZONE_C;
+import static net.ripe.db.whois.common.rpsl.AttributeType.*;
```

Comment:
use explicit imports (not star)

Reply:
oops.. seems like when i use shortcut to remove unused imports Intellij replaces the imports with *, even after setting prefrences :(